geminimd/util.py

```
@@ -0,0 +1,108 @@
+
+# Simple cache for discovered Drive IDs
+STATE_PATH = Path.home() / ".gmd_state.json"
+RUNS_PATH = Path.home() / ".gmd_runs.json"
+
+
+def _load_state() -> dict:
+    try:
+        if STATE_PATH.exists():
+            return json.loads(STATE_PATH.read_text(encoding="utf-8"))
+    except Exception:
+        pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)